### PR TITLE
move from nkyma to k3d with modular kyma

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:

- move from nkyma to k3d with modular kyma,
- apply serverless module via moduleTemplate,
- apply istio module via operator.

**Related issue(s)**

### CODE REVIEW

File: tests/scripts/kyma-dashboard-integration.sh [hunk 1]
```diff
@@ -6,26 +6,33 @@ export NO_COLOR=1
 export KUBECONFIG="$GARDENER_KYMA_PROW_KUBECONFIG"
```
Comment (halamix2) [inline tests/scripts/kyma-dashboard-integration.sh:6]: Gardener leftover

**Review by akucharska — APPROVED** [review]

**Pull request merged**